llvm/test/tools/llvm-lib/arm64ec-implib.test

```diff
@@ -98,6 +98,208 @@ RUN: llvm-lib -machine:arm64ec test.lib -out:test2.lib
 RUN: llvm-nm --print-armap test2.lib | FileCheck -check-prefix=ARMAP %s
 
 
+RUN: llvm-lib -machine:arm64ec -def:test.def -defArm64Native:test.def -out:testx.lib
+
+RUN: llvm-nm --print-armap testx.lib | FileCheck -check-prefix=ARMAPX %s
```

Comment:
What does `ARMAPX` stand for here?

Reply:
ARMAP part was picked after --print-armap llvm-nm argument, I guess it's "archive map". The X suffix is just something different than other tests and matches picked file name suffix. The inspiration to pick X was ARM64X (although that may not be the best choice, as mentioned in the earlier comment).